packager/media/formats/mp2t/ac3_header.cc

```diff
@@ -103,14 +103,28 @@ size_t Ac3Header::GetHeaderSize() const {
   return 0;
 }
 
-size_t Ac3Header::GetFrameSize() const {
+/// Calculate the size of the frame from the sample rate code and the 
+/// frame size code.
+/// @return the size of the frame (header + payload).
+size_t CalcFrameSize(uint8_t fscod, uint8_t frmsizecod) {
   const size_t kNumFscode = arraysize(kAc3SampleRateTable);
-  DCHECK_LT(fscod_, kNumFscode);
-  DCHECK_LT(frmsizecod_, arraysize(kFrameSizeCodeTable));
+  DCHECK_LT(fscod, kNumFscode);
+  DCHECK_LT(frmsizecod, arraysize(kFrameSizeCodeTable));
   // The order of frequencies are reversed in |kFrameSizeCodeTable| compared to
   // |kAc3SampleRateTable|.
-  const int index = kNumFscode - 1 - fscod_;
-  return kFrameSizeCodeTable[frmsizecod_][index] * 2;
+  const int index = kNumFscode - 1 - fscod;
+  return kFrameSizeCodeTable[frmsizecod][index] * 2;
+}
```

Comment:
Can you move it to the anonymous namespace above, i.e. after line 42?

Reply:
Sure can!